numba/core/types/scalars.py

```diff
@@ -38,10 +38,32 @@ def __init__(self, name, bitwidth=None, signed=None):
         self.bitwidth = bitwidth
         self.signed = signed
 
+    @property
+    def bitwidth(self):
+        return self.__bitwidth
+
+    @bitwidth.setter
+    def bitwidth(self, bitwidth):
```

Comment:
You can leave out defining the setter and have the `__init__` define the `self.__bitwidth` directly.

Reply:
removed the setters in ec4fe66